pkg/sink/sink_test.go

```diff
@@ -365,20 +364,35 @@ func TestHandleEvent(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error creating Post request: %s", err)
 	}
+
 	if resp.StatusCode != http.StatusCreated {
 		t.Fatalf("Response code doesn't match: %v", resp.Status)
 	}
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		t.Fatalf("Error reading response body: %s", err)
-		return
 	}
-	if !strings.Contains(string(body), "event (EventID: ") {
-		t.Errorf("Response body doesn't have eventid")
+
+	respData := struct {
```

Comment:
This is something that a lot of clients are also going to want to do. We should probably just make this an explicit type. This will also let us set JSON tags to set key values (e.g. "event_listener" vs "EventListener")